source.extension.vsixmanifest

```diff
@@ -7,18 +7,18 @@
         <Description xml:space="preserve">Progressive Scroll, updated to work with VS2017.</Description>
     </Metadata>
     <Installation>
-        <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[15.0]" />
-        <InstallationTarget Id="Microsoft.VisualStudio.Pro" Version="[15.0]" />
-        <InstallationTarget Version="[15.0]" Id="Microsoft.VisualStudio.Enterprise" />
-        <InstallationTarget Version="[15.0]" Id="Microsoft.VisualStudio.IntegratedShell" />
+        <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[15.0, 16.0)" />
+        <InstallationTarget Id="Microsoft.VisualStudio.Pro" Version="[15.0, 16.0)" />
+        <InstallationTarget Version="[15.0, 16.0)" Id="Microsoft.VisualStudio.Enterprise" />
+        <InstallationTarget Version="[15.0, 16.0)" Id="Microsoft.VisualStudio.IntegratedShell" />
     </Installation>
     <Dependencies>
-        <Dependency d:Source="Installed" Id="Microsoft.VisualStudio.MPF" DisplayName="Visual Studio MPF" Version="[10.0,15.0]" />
+        <Dependency d:Source="Installed" Id="Microsoft.VisualStudio.MPF" DisplayName="Visual Studio MPF" Version="[10.0,16.0)" />
     </Dependencies>
     <Assets>
         <Asset Type="Microsoft.VisualStudio.MefComponent" d:Source="Project" d:ProjectName="%CurrentProject%" Path="|%CurrentProject%|" TargetVersion="[15.0]" />
```

Comment:
I assume it's also necessary to fix [15.0] here?

Reply:
It looks kind of unreasonable, but it did enable me to install it on newest VS 15.6 enterprise edition after this modification.